cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -34,19 +65,90 @@ def schedule(schedule_data, appliance):
         schedule.delete()
 
 
-@pytest.mark.sauce
-@pytest.mark.tier(3)
-@test_requirements.report
-def test_schedule_queue(schedule, appliance):
+@pytest.mark.parametrize("interval", TIMER)
+def test_schedule_queue(schedule, appliance, interval):
     """ To test scheduling of report using options: Once, Hourly, Daily, Weekly, Monthly
 
     Polarion:
         assignee: pvala
         casecomponent: Reporting
-        caseimportance: high
         initialEstimate: 1/10h
+        tags: report
     """
 
     schedule.queue()
     view = schedule.create_view(ScheduleDetailsView)
-    view.flash.assert_message('The selected Schedule has been queued to run')
+    view.flash.assert_message("The selected Schedule has been queued to run")
+
+
+@pytest.mark.ignore_stream("5.9")
+@pytest.mark.parametrize("email", INVALID_EMAILS)
+def test_report_schedules_invalid_email(appliance, schedule_data, email):
+    """
+    This test case checks if invalid emails are accepted while creating a schedule
+    TODO: In addition to above patterns, there are few invalid patterns which are still accepted.
+    Patterns such as: xyz@example@example.com, xyz@example, ?/><!$%@example.com
+    BZ(1684491) has been filed for this.
+
+    Bugzilla: 1684491
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: report
+    """
+    schedule_data["emails"], schedule_data["from_email"] = (
+        INVALID_EMAILS[email],
+        INVALID_EMAILS[email],
+    )
+    with pytest.raises(AssertionError):
+        appliance.collections.schedules.create(**schedule_data)
+    view = appliance.collections.schedules.create_view(NewScheduleView)
+    view.flash.assert_message("One of e-mail addresses 'To' is not valid")
+    view.flash.assert_message("E-mail address 'From' is not valid")
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(server_roles="+notifier")
+def test_reports_create_schedule_send_report(smtp_test, schedule):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        startsin: 5.8
+        tags: report
+        setup:
+            1. Navigate to Cloud > Intel > Reports > Schedules.
+            2. Click on `Configuration` and select `Add a new Schedule`.
+            3. Create schedule that send an email to more than one users.
+            Un-check "Send if Report is Empty" option.
+        testSteps:
+            1. Queue up this Schedule and check if the email was sent.
+        expectedResults:
+            1. Queueing the schedule must send the report via email to all the users.
+    """
+    schedule.queue()
+    emails_sent = ",".join(schedule.emails)
+
+    # wait for emails to appear
+    wait_for(lambda: len(smtp_test.get_emails()) > 0, num_sec=90, delay=5)
```

Comment:
MyView:
@valaparthvi totally optional but I think we can take initial count and then go for wait
```
initial_count = len(smtp_test.get_emails(to_address=emails_sent))

try:
    wait_for(lambda: len(smtp_test.get_emails(to_address=emails_sent)) > initial_count, num_sec=90, delay=5)
except TimeoutError:
    assert False
```